data/data/vsphere/master/main.tf

```diff
@@ -30,8 +30,9 @@ resource "vsphere_virtual_machine" "vm" {
   }
 
   extra_config = {
-    "guestinfo.ignition.config.data"          = base64encode(data.ignition_config.ign[count.index].rendered)
+    "guestinfo.ignition.config.data"          = base64encode(var.ignition)
     "guestinfo.ignition.config.data.encoding" = "base64"
+    "guestinfo.hostname"                      = "${var.cluster_id}-${var.name}-${count.index}"
```

Comment:
should we also set this for bootstrap host. it wouldn't be applied today.. but maybe when rhcos does support that by default it would be nice.

Reply:
Sure I can update it to be set